contrib/helm/harbor/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: harbor
+version: 0.0.1
+appVersion: 1.3.0
+description: VMWare Enterprise Grade Docker Registry
+keywords:
+- vmware
+- docker
+- registry
+home: https://github.com/vmware/harbor
+icon: https://github.com/vmware/harbor/blob/master/docs/img/harbor_logo.png
+sources:
+- https://github.com/vmware/harbor
+maintainers:
+- name: paulczar
+  email: username.taken@gmail.com
```

Comment:
Is username.taken@gmail.com your personal email ? If yes, it is really great!

Reply:
yes